FordLabs/peoplemover
# Move isDragging state from redux to recoil

## Issue
Resolves N/A

## What was done
- [x] Move isDragging state from redux to recoil

## How to test
1. Ensure drag and drop works as it did before (Dragging a person from product to product, or to and from the unassigned section)
